.github/workflows/build.yml

```diff
@@ -48,15 +52,33 @@ jobs:
       with:
         name: lint_debug_log
         path: artifacts/lint_debug_log.log
+    - name: Create ID Set
+      run: |
+        demisto-sdk create-id-set -o artifacts/pack_id_set.json
+
+        gsutil -m cp -r gs://marketplace-dist/content/private_id_set.json artifacts/private_id_set.json
+        gsutil -m cp -r gs://marketplace-dist/content/id_set.json artifacts/id_set.json
+
+        demisto-sdk merge-id-sets -i1 artifacts/private_id_set.json -i2 artifacts/id_set.json -o artifacts/id_set.json
+        demisto-sdk merge-id-sets -i1 artifacts/pack_id_set.json -i2 artifacts/id_set.json -o artifacts/id_set.json
+
+    - name: Upload ID set file to artifacts
+      if: always()
+      uses: actions/upload-artifact@v1
+      with:
+        name: id_set
+        path: artifacts/id_set.json
     - name: Validate all packs
       run: |
-        demisto-sdk validate -a
+        git fetch
+        git branch
```

Comment:
still unsure why these lines are needed

Reply:
There is a new test in sdk that checks if the price of the premium packs has changed, this test is according to the metadata file that exists in the master of the pack -
For this test, we need to add these lines (without them it will fail).
I think we'll have to add it to private build too, @yaakovi am I right?